guides/common/modules/ref_notification-types.adoc

```diff
@@ -12,5 +14,7 @@ It contains a summary of errata applicable and installable to hosts registered t
 This allows a user to monitor what updates have been applied to which hosts.
 * *Puppet error state*: A notification sent after a host reports an error related to Puppet.
 * *Puppet summary*: A summary of Puppet reports.
+* *Repository sync failure*: A notification sent after repository synchronization fails.
+* *{SmartProxy} sync failure*: A notification sent after {SmartProxy} synchronization fails.
```

Comment:
If we are ordering this list alphabetically, then you need to place to this at different places depending on the build:
```diff
bef58c6bccdb71933c94e2e932fbe44cc3087c0e Commit
diff --git a/guides/common/modules/ref_notification-types.adoc b/guides/common/modules/ref_notification-types.adoc
index e8c0366e8..9efa698c2 100644
--- a/guides/common/modules/ref_notification-types.adoc
+++ b/guides/common/modules/ref_notification-types.adoc
@@ -4,17 +4,25 @@
 The following are the notifications created by {Project}:
 
 * *Audit summary*: A summary of all activity audited by {ProjectServer}.
+ifdef::satellite[]
+* *{SmartProxy} sync failure*: A notification sent after {SmartProxy} synchronization fails.
+endif::[]
 * *Content view promote failure*: A notification sent after content view promotion fails.
 * *Content view publish failure*: A notification sent after content view publication fails.
 * *Host built*: A notification sent when a host is built.
 * *Host errata advisory*: A summary of applicable and installable errata for hosts managed by the user.
 * *OpenSCAP policy summary*: A summary of OpenSCAP policy reports and their results.
+ifdef::orcharhino[]
+* *{SmartProxy} sync failure*: A notification sent after {SmartProxy} synchronization fails.
+endif::[]
 * *Promote errata*: A notification sent only after a Content View promotion.
 It contains a summary of errata applicable and installable to hosts registered to the promoted Content View.
 This allows a user to monitor what updates have been applied to which hosts.
 * *Puppet error state*: A notification sent after a host reports an error related to Puppet.
 * *Puppet summary*: A summary of Puppet reports.
 * *Repository sync failure*: A notification sent after repository synchronization fails.
+ifdef::foreman-el,foreman-deb,katello[]
 * *{SmartProxy} sync failure*: A notification sent after {SmartProxy} synchronization fails.
+endif::[]
 * *Sync errata*: A notification sent only after synchronizing a repository.
 It contains a summary of new errata introduced by the synchronization.
```

Feel free to use this patch, although you will need to resolve merge conflict caused by #2598.

Reply:
That's a very good point on how to order the list. Ordering it alphabetically would normally make sense, but I don't think it's worth the maintenance burden of three ifdefs (the same description being included three times, the risk of someone accidentally deleting an endif which asciidoctor doesn't report...).

Another option would be to look into how the notification types are ordered in the Web UI and mirror that.

I'll investigate.

Reply:
I decided to go with the order in Web UI.